core/src/main/java/org/apache/gravitino/storage/relational/mapper/StatisticMetaMapper.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.storage.relational.mapper;
+
+import java.util.List;
+import org.apache.gravitino.storage.relational.po.StatisticPO;
+import org.apache.ibatis.annotations.DeleteProvider;
+import org.apache.ibatis.annotations.InsertProvider;
+import org.apache.ibatis.annotations.Param;
+import org.apache.ibatis.annotations.SelectProvider;
+import org.apache.ibatis.annotations.UpdateProvider;
+
+public interface StatisticMetaMapper {
+
+  String STATISTIC_META_TABLE_NAME = "statistic_meta";
+
+  @SelectProvider(type = StatisticSQLProviderFactory.class, method = "listStatisticPOsByEntityId")
+  List<StatisticPO> listStatisticPOsByEntityId(@Param("entityId") long entityId);
```

Comment:
why don't you need a `metalakeId` as a filter condition? User should always list statistics under a specified metalake, right?

Reply:
EntityId is the uuid. I think it's enough.

Reply:
Then why does your schema design include the `metalakeId` field? 

Currently, all entities use uuid, but during list operations, `metalakeId` is used as the filter condition. I think we should align this behavior.

Reply:
The `metalakeId` is to make delete `metalake`  operations convenient. Ok, I can align the behaviour.